src/styles/App.css

```diff
@@ -12115,4 +12115,14 @@ body.sidebar-toggled footer.sticky-footer {
 [data-tip]:hover:before,
 [data-tip]:hover:after {
 	display:block;
-}
\ No newline at end of file
+}
+
+#userDropdown{
+
+color:rgba(108, 122, 137, 1);
```

Comment:
Please add proper spacing. Missing a tab here?

Reply:
should be fixed now